udata/commands/fixtures.py

```diff
@@ -39,7 +39,7 @@
 DISCUSSION_URL = "/api/1/discussions"
 
 
-DEFAULT_FIXTURE_FILE_TAG: str = "v2.0.0"
+DEFAULT_FIXTURE_FILE_TAG: str = "refs/heads/chore/update-org-badges-kind"
```

Comment:
Use fixtures with badges kind updated. See https://github.com/opendatateam/udata-fixtures/pull/5
We should validate and upgrade the fixtures to a new tag before merging this PR